TemplateApplication/Onboarding/OnboardingFlow.swift

```diff
@@ -17,12 +17,12 @@ import SwiftUI
 /// Displays an multi-step onboarding flow for the Spezi Template Application.
 struct OnboardingFlow: View {
     @Environment(HealthKit.self) private var healthKit
-
+    
     @Environment(\.scenePhase) private var scenePhase
     @Environment(\.notificationSettings) private var notificationSettings
-
+    
     @AppStorage(StorageKeys.onboardingFlowComplete) private var completedOnboardingFlow = false
```

Comment:
not directly relevant to this PR, but we have a nice strongly-typed `@LocalPreference` API in MHC (which is an alternative to `@AppStorage` that adds strong typing, consistent default values, and a couple other benefits. we might want to at some point put this into either SpeziFoundation or SpeziViews, so that it can be used in other applications as well...

Reply:
Added an issue for this at https://github.com/StanfordSpezi/SpeziViews/issues/66